src/tools/perf/lib/libperf.c

```diff
@@ -459,13 +459,17 @@ static ucs_status_t uct_perf_test_check_capabilities(ucx_perf_params_t *params,
 
     status = uct_md_query(md, &md_attr);
     if (status != UCS_OK) {
-        ucs_error("Failed to uct_md_query: %s", ucs_status_string(status));
+        ucs_error("uct_md_query(%s/%s) failed: %s",
+                  params->uct.tl_name, params->uct.dev_name,
```

Comment:
need md name, not tl/dev name

Reply:
> need md name, not tl/dev name

can use directly `md->component->md_name`?

Reply:
no
need to save it after uct_md_open is called
also there is no md_name in uct_component_t, its component name not md name

Reply:
thanks, done